src/components/link-wrapper.ts

```diff
@@ -2,9 +2,9 @@ import styled from "@emotion/styled";
 import { spacing } from "../constants";
 
 export const LinkWrapper = styled("div")`
-    display: grid;
-    justify-self: center;
-    justify-content: center;
-    width: 100%;
-    gap: ${spacing[1]};
-`;
+    a {
```

Comment:
if link wrapper is just stying an a tag, then this should be a styled link instead, no need for the div
```
export const LinkWrapper = styled("a")`
 ... styles
`;
```